Zend/tests/partial_application/magic_003.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+Partial application magic trampoline release
+--FILE--
+<?php
+class Foo {
+    static function __callStatic($name, $args) {}
+}
+Foo::method(?);
```

Comment:
This doesn't seem to fully fix the issue. The assertion is back if write `$x = Foo::method(?)` here.